build-notifier/send_email.py

```diff
@@ -0,0 +1,35 @@
+import argparse
+import smtplib
+from google.cloud import secretmanager
+
+
+arg_parser = argparse.ArgumentParser()
+arg_parser.add_argument(
+  '--email', help='Receiver email', type=str, required=True)
```

Comment:
I am curious why this can fetch the email directly.

Reply:
git clone and git log should show the author email as it worked locally. I guess the github action has some weird permission setup.

Reply:
Note the email is fetched here: https://github.com/datacommonsorg/website/pull/290/files#diff-83d8e1ea99371365d0e6f1fe066d4645R54-R57